.github/workflows/ci.yml

```diff
@@ -3,18 +3,19 @@ name: CI
 on:
   push:
     branches:
-      - '*'
-      - '!main'
+      - "*"
+      - "!main"
 
 env:
   BUILDER_VERSION: v0.8.11
   BUILDER_SOURCE: releases
   BUILDER_HOST: https://d19elf31gohf1l.cloudfront.net
-  PACKAGE_NAME: aws-crt-nodejs
```

Comment:
hmmm, I think I have the branch that only has the sigv4a stuff in it and those CI related stuff was removed [here](https://github.com/awslabs/aws-crt-nodejs-staging/pull/5/files). So, I guess we can squash that PR and cherry-pick the squashed commit.

Reply:
Or just remove all those changes in ci.yml, they are not needed for public repo. Either way.